celery_rpc/tests/utils.py

```diff
@@ -1,14 +1,28 @@
 # coding: utf-8
 from autofixture import AutoFixture
 from django.core.exceptions import ValidationError
+from rest_framework import serializers
 from celery_rpc.tests.models import SimpleModel
 from celery_rpc import utils
+from celery_rpc.base import DRF3
+
 
 def get_model_dict(model):
```

Comment:
`DRF>=3.0` with serializer.data returns string representations of datetime, while `DRF<3.0` returns python objects, ready for json dump. It's a change of contract, and serializer doesn't have methods to return all field values as python objects. `serializer.to_internal_value` seems to be ok, but it excludes read-only fields (PK) and fields without defaults.